linux-anvil-cuda/Dockerfile

```diff
@@ -38,7 +30,14 @@ RUN yum update -y && \
         sudo \
         tar \
         which && \
-    yum clean all
+    : '(centos:6 only) If glibc-common is updated, we have to manually purge non-en_US locales.' && \
```

Comment:
Docker doesn't like real comments as part of a multi-line `RUN` so we have to work around that if we want them.
(Ab-)using the null utility (special built-in) `:` for this is a little hacky, TBH, but it *really* doesn't do any harm, so it is perfectly fine in my book.

Reply:
Could we move this out into a bash script that we add and run to all `Dockerfile`s? This will ensure we only need to update and maintain this in one spot (as opposed to every Docker image), which will keep things maintainable here. Also we can format this more nicely without having to deal with Docker's syntax oddities. 🙂

Reply:
Sure, I had the same thing in mind. I was just in the "meh, I'll likely refactor stuff later anyway when proposing multi-staged builds, so whatever" state :wink: .

Reply:
Now it uses a common script in all images so that the `rm -rf /var/cache/yum/*` addition is only in one place, too.